salt/container-feeder/init.sls

```diff
@@ -9,7 +9,7 @@ include:
 container-feeder:
   service.running:
```

Comment:
Since it's a one-shot service maybe we can completely remove `reload` and `restart` from here. If that makes `container-feeder` run it'll be fine. Also, I see related past issues (https://github.com/saltstack/salt/issues/37842) with one shot services, so worst case for `container-feeder` we replace this with a plain `cmd.run` that runs `systemctl start container-feeder`.

Reply:
I think restart was the original intent  - if CF was started before docker, it could be failing out, when we restart docker, we need to restart CF to be sure it starts clean from the beginning.

Reply:
I think the magic is on the `mod_watch` on `service.py`: https://github.com/saltstack/salt/blob/2016.11/salt/states/service.py#L596-L615

It should be `full_restart`. Even if we dropped to `cmd.run` it should be ran after docker has been (re)started since it's requiring it and also watching it.